kyaml/fn/runtime/runtimeutil/functiontypes.go

```diff
@@ -75,6 +161,12 @@ type ContainerSpec struct {
 
 	// User is the username/uid that application runs as in continer
 	User ContainerUser `json:"user,omitempty" yaml:"user,omitempty"`
+
+	// EnvRaw is a slice of env string.
+	EnvRaw []string `json:"envs,omitempty" yaml:"envs,omitempty"`
```

Comment:
Provide a better comment.  Is this to provide a means to serialize the contents of ContainerEnv?

Reply:
Yes, it's used to serialize/deserialize envs from function configs.

Reply:
We shouldn't have two representations of the same data.

Drop this slice, and add field tags to ContainerEnv instead.

Examples: https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/yaml/types.go#L166

Reply:
But the envs should be a slice of string in the function config yaml. Can I serialize the ContainerEnv struct into a slice of string by tag?